### ISSUE TITLE
Abnormal Letter-Spacing misplaces caret

### ISSUE THREAD

**Issue by ultimate-tester** [kazzkiq/CodeFlask#109]
Quite simple problem, having an abnormal letter-spacing will prevent the caret aligning properly. The CSS used by CodeFlash should reset the letter-spacing to the default, which is `letter-spacing: normal;`

![image](https://user-images.githubusercontent.com/580758/74981058-e0cec780-5431-11ea-8a05-e5807b7a4ea8.png)

**Comment by brentkelly** [comment]
See below - this same problem exists with a stock standard markdown highlighted example. Problem persists even when setting `letter-spacing: normal;` for all elements.

![example](https://i.imgur.com/aQgWGjY.png)

See the following pen: https://codepen.io/brentkelly/pen/gOabdmQ

NB: https://github.com/ClickSimply/nanoEditor doesn't seem to suffer this issue.

**Comment by kazzkiq** [comment]
CodeFlask works by syncing textarea text with highlighted editor text. So any differences in `letter-spacing` (or any other font or spacing CSS properties) will misplace caret.

In your case you changed `letter-spacing` in your project CSS on purpose? Or was it a side-effect of a CSS meant to style other part of your project?

**Comment by ultimate-tester** [comment]
> 
> 
> CodeFlask works by syncing textarea text with highlighted editor text. So any differences in `letter-spacing` (or any other font or spacing CSS properties) will misplace caret.
> 
> In your case you changed `letter-spacing` in your project CSS on purpose? Or was it a side-effect of a CSS meant to style other part of your project?

Thanks for responding. In my case it was the latter. Other CSS in my project caused this as a side effect. My advice is to reset the letter-spacing (or take it into account) for CodeFlask.

**Comment by kazzkiq** [comment]
Yep. But the issue is that other styles can still leak into CodeFlask editor. Is basically a non-ending nightmare when you run without sandboxed code.

I believe this kind of issue will vanish with #119.
